_includes/google_analytics.html

```diff
@@ -1,13 +1,11 @@
 {% if site.google_universal_analytics %}
-  {% raw %}
```

Comment:
I had to remove this tag, is this the best approach the issue I mentioned in the comments? It seems good to me.